content/en/docs/tasks/administer-cluster/cluster-management.md

```diff
@@ -189,18 +189,6 @@ node discovery; currently this is only Google Compute Engine, not including Core
 
 ## Advanced Topics
 
-### Upgrading to a different API version
-
-When a new API version is released, you may need to upgrade a cluster to support the new API version (e.g. switching from 'v1' to 'v2' when 'v2' is launched).
-
-This is an infrequent event, but it requires careful management. There is a sequence of steps to upgrade to a new API version.
-
```

Comment:
@missingcharacter . Thank for opening this pull request.
Looking at the change, if these steps are removed, what are the replacement steps?
@caesarxuchao Would you have time to review? Thanks!